primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.commandlink.js

```diff
@@ -79,16 +79,15 @@ PrimeFaces.widget.CommandLink = PrimeFaces.widget.BaseWidget.extend({
     disable: function() {
         this.jq.removeClass('ui-state-hover ui-state-focus ui-state-active')
                 .addClass('ui-state-disabled')
-                .attr('tabindex', '-1')
-                .attr('aria-disabled', 'true');
+                .attr('tabindex', '-1');
```

Comment:
Don't you want to leave these so if they use the widget to control disabled?

Reply:
There was a warning that you should not specify it on an anchor with an `href` attribute.

Reply:
Oh OK